src/features/dashboard/ui/Checklist.tsx

```
@@ -4,6 +4,10 @@ import { getMenuLists } from '../../../shared/types/dashboardType';
 import { useParams } from 'react-router-dom';
 // import completeCheck from '../../../../public/assets/dashboard/main/Check(complete).svg';
 
+const checkLists = menuLists
```

ESLint

[error] 'checkLists' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)